Helper/Cart.php

```diff
@@ -2208,7 +2208,7 @@ public function collectDiscounts(
                         case RuleInterface::COUPON_TYPE_NO_COUPON:
                         default:
                             $discounts[] = [
-                                'description'       => trim(__('Discount ') . $rule->getDescription()),
+                                'description'       => trim(__('Discount ') . $rule->getName()),
```

Comment:
This maybe too big change. Have you faced an issue when description is empty? How about $rule->getDescription() ?? $rule->getDescription() ?

Reply:
`$rule->getDescription() ?? $rule->getName()`

Reply:
@danac-gs @vitaliyreznikov Updated the code